client/src/app.scss

```
@@ -21,36 +22,55 @@
   --font-heading: Inter, var(--font-fallback);
   --font-code: Menlo, Consolas, Monaco, Liberation Mono, Lucida Console,
     monospace;
-  --font-size: 16px;
-  --font-line-height: 1.2;
+
+  --base-font-size: 100%;
+  --base-line-height: 1.2;
+  --intermediate-line-height: 1.5;
+  --heading-line-height: var(--base-line-height);
+  --heading-letter-spacing: -0.031rem;
+  --font-content-line-height: 1.75;
   --font-body-strong-weight: 600;
 
-  // defined in Figma here: https://www.figma.com/file/cAd46zJb6H7mW7W14iLsDe/MDN-Basic-Components?node-id=299%3A311
-  --type-heading-l: 500 1rem/1.1875 var(--font-heading); /* 16px/19px, h5 */
-  --type-title-l: 400 1.5rem/1.2 var(--font-heading);
-  --type-title-s: var(--font-body-strong-weight) 0.8125rem/1.2 var(--font-body);
-  --type-body-l: 400 1rem/1.75 var(--font-body); /* 16px/28px */
-  --type-body-m: 400 0.8125rem/1.23 var(--font-body); /* 13px/16px */
-  --type-body-s: 500 0.6875rem/1.273 var(--font-body); /* 11px/14px */
-  --type-emphasis-l: var(--font-body-strong-weight) 1rem/1.1875 var(--font-body); /* 16px/19px */
-  --type-emphasis-m: var(--font-body-strong-weight) 0.8125rem/1.23
-    var(--font-body); /* 13px/16px */
-  --type-label-s: 600 0.625rem/1.2 var(--font-heading); /* 10px/12px */
-  --type-code-regular: 400 0.875rem/1.5 var(--font-code); /* 14px/21px */
-  --type-code-inline: 400 0.875em/1.5 var(--font-code); /* 14px/21px */
-
-  // From Figma, here: https://www.figma.com/file/7lbOm57iOjSVIGORzXqrlW/MDN-Articles?node-id=1%3A27
-  --type-heading-h1: 300 2.25rem/1.138 var(--font-heading); /* 36px/41px */
-  --type-heading-h2: 400 1.75rem/1.2857 var(--font-heading); /* 28px/36px */
-  --type-heading-h3: 500 1.5rem/1.2916 var(--font-heading); /* 24px/31px */
-  --type-heading-h4: 500 1.125rem/1.222 var(--font-heading); /* 18px/22px */
-  --type-heading-h5: var(--type-heading-l);
-  --type-heading-h6: 500 0.75rem/1.0833 var(--font-heading); /* 12px/13px */
-  --type-article-p: 400 1rem/2 var(--font-body); /* 16px/200% */
+  /* based on the following type-scale https://type-scale.com/?size=16&scale=1.200&text=MDN%20Web%20Docs&font=Inter&fontweight=400&bodyfont=body_font_default&bodyfontweight=400&lineheight=1.75&backgroundcolor=%23ffffff&fontcolor=%23000000&preview=false */
+  --type-heading-h1-font-size: 2.488rem;
+  --type-heading-h2-font-size: 2.074rem;
+  --type-heading-h3-font-size: 1.728rem;
+  --type-heading-h4-font-size: 1.44rem;
+  --type-heading-h5-font-size: 1.2rem;
+  --type-base-font-size-rem: 1rem;
+  --type-smaller-font-size: 0.833rem;
+  --type-tiny-font-size: 0.694rem;
```

I think it would be better if these variables shared a common prefix (and I'm not sure what the `type-` means):

```suggestion
  --font-size-smaller: 0.833rem;
  --font-size-tiny: 0.694rem;
```

---

I probably meant to call it `typography` no idea why I did not 😆

---

Ah no, I see. This was the convention already in place so I just followed. I believe something like what you suggest here is better but, perhaps it can be a cleanup post this merge? I can change it now but, it will touch potentially even more files than what this already touches.

---

Suggestion from outsider: for now, change from `--type-smaller-font-size` and `--type-tiny-font-size` to `--type-font-size-smaller` and `-type-font-size-tiny`. Remove `type` in a follow-up PR.

Current uses of this convention do put the common word first (e.g. `--type-code-regular` and `--type-code-inline`)

---

It's better than before, so I will resolve this thread for now.